### PR TITLE
Integration Tests groundwork

### PR DESCRIPTION
**Pull request by spg**

### CODE REVIEW

**Comment by meriouma** [comment]
Nice! LGTM

**Comment by olafleur** [comment]
Cool! I didn't know AssertJ. LGTM.

File: integration-test/src/main/java/com/arcbees/chosen/integrationtest/client/CarBrand.java [hunk 1]
```diff
@@ -0,0 +1,21 @@
+package com.arcbees.chosen.integrationtest.client;
```
Comment (christiangoudreau) [inline integration-test/src/main/java/com/arcbees/chosen/integrationtest/client/CarBrand.java]: missing license header

File: integration-test/src/main/java/com/arcbees/chosen/integrationtest/client/ChosenSampleIntegrationTests.java [hunk 2]
```diff
@@ -0,0 +1,27 @@
+package com.arcbees.chosen.integrationtest.client;
```
Comment (christiangoudreau) [inline integration-test/src/main/java/com/arcbees/chosen/integrationtest/client/ChosenSampleIntegrationTests.java]: Idem (All new Java files)

File: integration-test/src/test/java/SampleIT.java [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class SampleIT {
+    private static final String ROOT = "http://localhost:8080";
+
+    private final WebDriver webDriver = new ChromeDriver();
+    private static final int TIME_OUT_IN_SECONDS = 20;
+
+    @Test
+    public void sampleTest() throws Throwable {
```
Comment (christiangoudreau) [inline integration-test/src/test/java/SampleIT.java]: Will we be using cucumber?

**Comment by jDramaix** [comment]
This is very nice @spg  !!!! Integration tests will be really useful for this project and will avoid regression !

**Comment by jDramaix** [comment]
and LGTM !

File: integration-test/src/test/java/SampleIT.java (hunk 3 above)
Comment (spg) [inline integration-test/src/test/java/SampleIT.java]: Maybe sometime in the future, we'll see if we need it as the test base grows.

File: integration-test/src/main/java/com/arcbees/chosen/integrationtest/client/ChosenSampleIntegrationTests.java (hunk 2 above)
Comment (spg) [inline integration-test/src/main/java/com/arcbees/chosen/integrationtest/client/ChosenSampleIntegrationTests.java]: @christiangoudreau  done

**Pull request merged**